Library/Homebrew/api.rb

```diff
@@ -18,7 +18,8 @@ module API
 
     module_function
 
-    API_DOMAIN = "https://formulae.brew.sh/api"
+    API_DOMAIN = Homebrew::EnvConfig.api_domain.freeze
+    API_DEFAULT_DOMAIN = HOMEBREW_API_DEFAULT_DOMAIN
```

Comment:
Should we remove these two variables and use `EnvConfig` directly instead?

Reply:
I think we can just remove this line and use `HOMEBREW_API_DEFAULT_DOMAIN` throughout this file. I don't think it looks out of place or inconsistent to have `API_DOMAIN` and `HOMEBREW_API_DEFAULT_DOMAIN`, even though they aren't quite parallel names